docs/en/ecosystem/external-table/iceberg-of-doris.md

```diff
@@ -135,10 +167,15 @@ You can also create an Iceberg table by explicitly specifying the column definit
     - The order of the columns needs to be consistent with the Iceberg table
 - ENGINE needs to be specified as ICEBERG
 - PROPERTIES property.
-    - `iceberg.hive.metastore.uris`: Hive Metastore service address
     - `iceberg.database`: the name of the database to which Iceberg is mounted
     - `iceberg.table`: the name of the table to which Iceberg is mounted, not required when mounting Iceberg database.
-    - `iceberg.catalog.type`: the catalog method used in Iceberg, the default is `HIVE_CATALOG`, currently only this method is supported, more Iceberg catalog access methods will be supported in the future.
+    - `iceberg.catalog.type`: the catalog type used in Iceberg, the default is `HIVE` (simple name of `org.apache.iceberg.hive.HiveCatalog`).
+    - `iceberg.catalog.catalog-impl`: you could also use the full catalog name with this property. You should not use both `iceberg.catalog.type` and `iceberg.catalog.catalog-impl`.
```

Comment:
The user does not need to care about the specific implementation classes of the catalog already supported in iceberg, which would make it more difficult to use. Such as `JDBC`/`Glue` and so on.

User only needs to set the catalog type and the specific implementation class Doris should encapsulate it.

Wouldn't it be better if the user only needs to set the implementation class information when using a custom catalog?

Reply:
Do you mean always set the full class name?  Would it be inconvenient for users who use the `HIVE` or `Hadoop`?

Reply:
> Do you mean always set the full class name? Would it be inconvenient for users who use the `HIVE` or `Hadoop`?

What I mean is that the catalog types already supported by iceberg can be set without setting the full class name, just set the type.
For custom catalog types, only the full class name needs to be set

Reply:
Updated